payments-ui-core/src/main/java/com/stripe/android/ui/core/forms/resources/AsyncResourceRepository.kt

```diff
@@ -20,15 +21,17 @@ import kotlin.coroutines.CoroutineContext
 @Singleton
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 class AsyncResourceRepository @Inject constructor(
-    private val resources: Resources?,
+    private val resources: Resources,
     @IOContext private val workContext: CoroutineContext,
     private val locale: Locale?
 ) : ResourceRepository {
+    private lateinit var lpmRepository: LpmRepository
```

Comment:
We add the LPM repository here, so that it can be later asynchronously loaded from file or server.

Reply:
Why does this have to be a lateinit var if we are just setting it in an init?

Reply:
done in error, fixing